topics/ecology/tutorials/obisindicators/tutorial.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: tutorial_hands_on
+
+title: Obis marine indicators
+zenodo_link: ''
+questions:
+objectives:
+- Calculating and vizualizing marine biodiversity indicators
+time_estimation: 1H
+key_points:
+- Marine data
+- Diversity indicators
+contributors:
+- Marie59
+- yvanlebras
+
+---
+
+
+
+# Introduction
+
+OBIS is a global open-access data and information clearing-house on marine biodiversity for science, conservation and sustainable development. In order to visualize their marine data OBIS created the package obisindicators.
+
+Obisindicators is an R library developed during the 2022 IOOS Code Sprint. The purpose was to create an ES50 diversity index within hexagonal grids following the diversity indicators notebook by Pieter Provoost linked above. The package includes several examples, limited to 1M occurrences, that demonstrate uses of the package.
+This tutorial will guide you on getting obis marine data and processing them in order to calculate and visualize multiple indicators. 
+
+This tool for obisindicators is composed of 5 indicators : Number of record, Shannon, Simpson, Es50 and Hill which will be explained in more details later on.
+
+In this tutorial, we'll be working on obis data extracted from .  First those data will be prepared. 
+
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+## Upload of the data
+
+This first step consist of downloading and uploading obis data onto galaxy.
+
+> <hands-on-title>Data upload</hands-on-title>
+>
+> 1. Create a new history for this tutorial and give it a name (example: “Obisindicators tutorial”) for you to find it again later if needed.
+>
+>    {% snippet faqs/galaxy/histories_create_new.md box_type="none" %}
```

Comment:
```suggestion
>    {% snippet faqs/galaxy/histories_create_new.md %}
```

this is intended to be a tip box (so experienced galaxy users can skip over it. Is there a reason to remove the box type here?

Reply:
No reason at all ! I think it's an unfortunate copy paste ;)